docs/development/client-code-generation.md

```diff
@@ -4,7 +4,7 @@ To enable a server to produce OpenApi spec, just include the following code bloc
 ```
 plugin {
   id 'com.github.johnrengelman.processes' version '0.5.0'
-  id 'org.springdoc.openapi-gradle-plugin' version '1.3.3'
+  id 'org.springdoc.openapi-gradle-plugin' version '1.6.0'
```

Comment:
So with this upgrade khttp lib is not required anymore?

Reply:
Yes.